examples/generator-fullstack/model/datamodel.graphql

```diff
@@ -1,21 +1,22 @@
 type Note {
   id: ID!
+  """
+  @db.type: 'string'
+  @db.length: 100
```

Comment:
I'm getting lost, are we using comment-style-annotations or inline-annotations `@OneToMany`?

Reply:
Myself and @wtrocki discussed this yesterday and agreed to go full comment-annotations in this PR. The inline directives are a legacy from the current Graphback approach and my plan was to do this in another PR (because this PR is already massive) but we agreed to do it in this PR.